CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v5.9.0 (1 July 2020)
+
+### Feature
+* Allow dynamic groups for group JEXL transform (15491285b4fb4fbd50c791c424ef5337f043cf46)
+
+### Documentation
+* Change wording in dynamic group docs (b09c3afdd899082d5ecfa6d07d67e1de1d8278e9)
```

Comment:
This has only been added in this release, so this doesn't help much, does it? Instead we could add a link to the new docs.

Reply:
Changed